src/main.cpp

```diff
@@ -1,6 +1,15 @@
 #include "stdafx.h"
+#include "config_parser.h"
 
 int main()
 {
-	return 0;
-}
+	std::ostringstream oss;
+	std::string fullLogName;
+	CConfigParser myConfig;
+
+	myConfig.ParseConfigFile(oss);

```

Comment:
It would be better when function can show if it fails or not. Like returning result etc..